### ISSUE TITLE
Can't load PNG files from chessmaniac.com

### ISSUE THREAD

**Issue by pychess** [pychess/pychess#64]
Original [issue 64](https://code.google.com/p/pychess/issues/detail?id=64) reported by [jjrussell](https://code.google.com/u/118247235144499884036/) 2006-10-31

**What steps will reproduce the problem?**
1. Load the attached file with PyChess
**2.**
**3.**

**What is the expected output? What do you see instead?**
The game history would be populated in the move history sidebar and you'd
be able to navigate through the game.

Instead the New Game dialog pops up.

**What version of the product are you using?**
0.4

**Please provide any additional information below.**

**Comment by pychess** [comment]
- attached [game1250281.pgn](https://code.google.com/p/pychess/issues/detail?id=64#c0)

**Comment by pychess** [comment]
Original [comment 1.](https://code.google.com/p/pychess/issues/detail?id=64#c1) posted by [thijsvermeir](https://code.google.com/u/101386254946966013986/) 2006-10-31

Thanks, for your bug report.

I confirmed this bug and fixed it. 
It didn't work because this pgn file used the long notation.
(1. e2-e4 c7-c5 2. c2-c4 g7-g6 3. d2-d4 ...)

The svn trunk version should be able to read this pgn file.
- changed status to: Fixed
- changed owner to: ---
